### PR TITLE
Mjau 460

### PR DESCRIPTION
**Pull request by skancharla8649**
- Tracking player and device volume

### CODE REVIEW

File: PulsePlayer/PulsePlayer/ios/SkinViewController.m [hunk 1]
```diff
@@ -314,6 +318,16 @@ - (IBAction)closeButtonPressed {
   [self dismissViewControllerAnimated:YES completion:nil];
 }
 
+- (IBAction)playerVolumeSliderChanged:(id)sender {
+    //    AVPlayer has no volume property and requires the use of the system volume setting which can be controlled only by the hardware switch or an MPVolumeView
+    //   volumecontrol reference- https://github.com/12Rockets/VolumeControl
+    //    MPVolumeView only works on an actual device and not on a simulator
+    MPVolumeView *volumeView = [[MPVolumeView alloc] init];
```
Comment (nichenriksson) [inline PulsePlayer/PulsePlayer/ios/SkinViewController.m]: `volumeView` doesn't seem to be used anywhere?

File: PulsePlayer/PulsePlayer/ios/SkinViewController.m [hunk 2]
```diff
@@ -314,6 +318,16 @@ - (IBAction)closeButtonPressed {
...
+    self.player.volume = _volumeSlider.value;
+    [_volumeSlider setContinuous: NO];
```
Comment (nichenriksson) [inline PulsePlayer/PulsePlayer/ios/SkinViewController.m]: `setContinuous` only needs to be set once AFAIK. Here it gets set to the same value every time the volume is changed, which seems unnecessary.

**Comment by nichenriksson** [comment]
The positioning of the volume slider is weird when I run it on an iPhone 12 mini simulator, it shows up in the middle of the ad.

**Comment by prve** [comment]
> The positioning of the volume slider is weird when I run it on an iPhone 12 mini simulator, it shows up in the middle of the ad.

@nichenriksson  This should have been fixed with commit: 
https://github.com/INVIDITechnologies/pulse-sdk-ios-2.x-sample/pull/9/commits/5994d2852417578aa14d04cf04e9540e90530d4d

Do you see any such issue after pulling changes in above commit?

**Comment by nichenriksson** [comment]
> > The positioning of the volume slider is weird when I run it on an iPhone 12 mini simulator, it shows up in the middle of the ad.
> 
> @nichenriksson This should have been fixed with commit:
> [5994d28](https://github.com/INVIDITechnologies/pulse-sdk-ios-2.x-sample/commit/5994d2852417578aa14d04cf04e9540e90530d4d)
> 
> Do you see any such issue after pulling changes in above commit?

Hmm, ok, I thought I had the latest code when testing. Will check again tomorrow.

**Comment by nichenriksson** [comment]
> > > The positioning of the volume slider is weird when I run it on an iPhone 12 mini simulator, it shows up in the middle of the ad.
> > 
> > 
> > @nichenriksson This should have been fixed with commit:
> > [5994d28](https://github.com/INVIDITechnologies/pulse-sdk-ios-2.x-sample/commit/5994d2852417578aa14d04cf04e9540e90530d4d)
> > Do you see any such issue after pulling changes in above commit?
> 
> Hmm, ok, I thought I had the latest code when testing. Will check again tomorrow.

Yes, you're right. Pulled the latest changes, now it looks great!

**Pull request closed without merging**